Doc/library/constants.rst

```diff
@@ -97,15 +97,17 @@ should not be used in programs.
           exit(code=None)
 
    Objects that when printed, print a message like "Use quit() or Ctrl-D
-   (i.e. EOF) to exit", and when called, raise :exc:`SystemExit` with the
+   (i.e. EOF) to exit", and when accessed directly in the interactive
+   interpreter or called as functions, raise :exc:`SystemExit` with the
    specified exit code.
 
 .. data:: help
    :noindex:
 
    Object that when printed, prints the message "Type help() for interactive
```

Comment:
And here.

Reply:
> Run `print(help)` and see what is returned

wow, my bad.

I didn't notice the difference between *printed* and *accessed directly*. Sorry about that.